doc/host-applications.rst

```diff
@@ -177,7 +177,7 @@ Finally, we compile it altogether:
 ::
 
     # spicyc -c my-http.spicy -o my-http.cc
-    # spicyc -l my-http.cc -o my-http-linker.cc
+    # spicyc -l my-http.spicy -o my-http-linker.cc
```

Comment:
This is just to get around the spicyc error message, right? 

I think we should still add a note on optimization, at least something along the lines of "if you compile multiple Spicy modules independently into C++ code, you will currently need to turn off global optimizations".